tools/test-tools/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # @fluidframework/test-tools
 
+## vNext
```

Comment:
I'm not aware of what our rules with the changelog is. I guess vNext is a placeholder that will be changed later?

Reply:
Yeah, and I don't know that we have a particular convention for this kind of thing, I went with my gut. Since we don't pre-bump these, I don't know if the next version will be 2.1.0 or 3.0.0.